x/ecocredit/location.go

```diff
@@ -16,7 +16,7 @@ var reLocation = regexp.MustCompile(`^([A-Z]{2})(?:-([A-Z0-9]{1,3})(?: ([a-zA-Z0
 func validateLocation(location string) error {
 	matches := reLocation.FindStringSubmatch(location)
 	if matches == nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "Invalid retirement location: %s.\nLocation should have format <country-code>[-<region-code>[ <postal-code>]].\n", location)
+		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "Invalid location: %s.\nLocation should have format <country-code>[-<region-code>[ <postal-code>]].\n", location)
```

Comment:
This error gets thrown sometimes for an invalid project_location, so I updated this to make it generic to project and/or retirement location.

Reply:
looks like a few test cases need to be updated for this